app/geo/gui/src/ts/twxplore/gui/geo/featureTypeStrategies/PointFeatureTypeStrategy/PointFeatureTypeStrategy.ts

```diff
@@ -1,7 +1,13 @@
 import {FeatureTypeStrategy} from "../FeatureTypeStrategy";
-import {FeatureType} from "../../api/graphqlGlobalTypes";
+import {FeatureType, FeatureQuery} from "../../api/graphqlGlobalTypes";
 
 export abstract class PointFeatureTypeStrategy implements FeatureTypeStrategy {
+  getQuery(clickedFeatureUri: string): FeatureQuery {
```

Comment:
This doesn't make much sense. A point can never have a within query. Return null to indicate no query should be issued?